server/test/repository/AccountRepositoryTest.java

```diff
@@ -75,24 +73,33 @@ public void updateClientNameTest_AllNonEmpty() {
     ApplicantModel applicantUpdateTest = setupApplicantForUpdateTest();
     AccountModel account = setupAccountForUpdateTest();
     repo.updateTiClient(
-        account, applicantUpdateTest, "Dow", "James", "John", "", "", "", "2020-10-10");
-    assertThat(applicantUpdateTest.getApplicantData().getApplicantFirstName().get())
-        .isEqualTo("Dow");
-    assertThat(applicantUpdateTest.getApplicantData().getApplicantMiddleName().get())
-        .isEqualTo("James");
-    assertThat(applicantUpdateTest.getApplicantData().getApplicantLastName().get())
-        .isEqualTo("John");
+        account,
+        applicantUpdateTest,
+        "Dow",
+        "James",
+        "John",
+        "1234567890",
+        "note",
+        "email@address.com",
+        "2020-10-10");
+    assertThat(applicantUpdateTest.getFirstName().get()).isEqualTo("Dow");
```

Comment:
suggestion (non-blocking): same suggestion as above. test that these values are set in both places for now.